Sources/ProjectDescription/Environment.swift

```diff
@@ -5,24 +5,6 @@ public struct Environment {
     public enum Value {
         case boolean(Bool)
         case string(String)
-
-        /// Retrieve the Environment value as a string or return the specified default string value
-        /// - Parameters:
-        ///   - default: default String value to be returned
-        /// - Returns: String
-        public func getString(default defaultString: String) -> String {
```

Comment:
can we just mark this `@deprecated`? Otherwise, this change would be breaking which is something we should try to avoid.

Reply:
Makes sense. Will do.